README.md

```diff
@@ -1,4 +1,4 @@
-![Build](https://github.com/Azure/bicep/workflows/Build/badge.svg)
+[![Build](https://github.com/Azure/bicep/actions/workflows/build.yml/badge.svg)](https://github.com/Azure/bicep/actions/workflows/build.yml)
```

Comment:
I think you need to keep the original image link "https://github.com/Azure/bicep/workflows/Build/badge.svg" here, so this line should be:
```markdown
[![Build](https://github.com/Azure/bicep/workflows/Build/badge.svg)](https://github.com/Azure/bicep/actions/workflows/build.yml)
```